src/coreclr/tools/Common/JitInterface/ThunkGenerator/InstructionSetDesc.txt

```diff
@@ -72,9 +72,12 @@ implication        ,X86   ,AES       ,SSE2
 implication        ,X86   ,BMI1      ,AVX
 implication        ,X86   ,BMI2      ,AVX
 implication        ,X86   ,FMA       ,AVX
+implication        ,X86   ,LZCNT     ,X86Base
 implication        ,X86   ,PCLMULQDQ ,SSE2
 implication        ,X86   ,POPCNT    ,SSE42
+implication        ,X86   ,Vector128 ,SSE
 implication        ,X86   ,Vector256 ,AVX
+implication        ,X86   ,AVXVNNI   ,AVX2
```

Comment:
Couple new "implications" for x86. Nothing major, just ensuring `AvxVnni` and `Lzcnt` are correctly dependent on the ISAs they inherit from and ensure `Vector128` is correctly marked as being dependent on `Sse`